dask/array/tests/test_creation.py

```diff
@@ -571,9 +571,15 @@ def test_repeat():
         assert all(concat(d.repeat(r).chunks))
 
 
+@pytest.mark.parametrize("a", [da.asarray([0, 1, 2]), [[1, 2], [3, 4]]])
```

Comment:
I have a slight preference to avoid creating dask array objects at import time.  This is probably fine, but I suspect that some future bugs might interfere more easily here.  Is it easy for you to bring this into the function?  Maybe in this one case we drop the parametrization.

Reply:
Thanks, dropped the parametrization.